connectors/http/rest/README.md

```diff
@@ -229,8 +229,9 @@ leading to the following result
 ```
 
 
-| Connector Info          |                        |
-|-------------------------|------------------------|
-| Type                    | io.camunda:http-json:1 |
-| Version                 | 8                      |
-| Supported element types |                        |
+| Connector Info            |                                                                       |
```

Comment:
@mathias-vandaele this change will cause the Spotless check to fail.

Please revert this one 😄